dts/arm/nxp/nxp_lpc55S3x_common.dtsi

```diff
@@ -0,0 +1,245 @@
+/*
```

Comment:
remove `label` property from all nodes in here

Reply:
Already removed it.